api/src/pcapi/core/users/api.py

```diff
@@ -1368,3 +1373,87 @@ def notify_users_before_deletion_of_suspended_account() -> None:
                 "Could not send email before deletion of suspended account",
                 extra={"user": account.id},
             )
+
+
+def anonymise_user(user: users_models.User, *, force: bool = False) -> None:
+    """
+    Anonymise the given User. If force is True, the function will anonymise the user even if it has an adress and we
+    cannot find an iris for it.
+    """
+    iris = None
+    if user.address:
+        try:
+            iris = get_iris_from_address(address=user.address, postcode=user.postalCode)
+        except geography_exceptions.AddressApiError as exc:
+            logger.error("Could not anonymise user %s: %s", user.id, exc)
```

Comment:
En général on met les variables en extra pour faciliter le filtrage :
```suggestion
            logger.error("Could not anonymize user", extra={"user_id":  %s, "exc": exc})
```